lib/client-assets.php

```diff
@@ -708,7 +708,8 @@ function gutenberg_extend_block_editor_settings_with_fse_theme_flag( $settings )
  * Sets the editor styles to be consumed by JS.
  */
 function gutenberg_extend_block_editor_styles_html() {
-	$handles = array(
```

Comment:
It feels important to either rename this function or split it up, as the name no longer reflects its purpose.